src/main/java/com/minecolonies/achievements/MineColoniesAchievement.java

```diff
@@ -0,0 +1,63 @@
+package com.minecolonies.achievements;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.stats.Achievement;
+
+/**
+ * A sublcass for this mod.
+ *
+ * May contain custom logic for achievements
+ *
+ * @author Isfirs
+ * @since 0.2
+ */
+public class MineColoniesAchievement extends Achievement
+{
+
+    /**
+     * Inherit parent constructor
+     * 
+     * @param id
+     * @param name
+     * @param offsetX
+     * @param offsetY
+     * @param itemIcon
+     * @param parent
+     */
+    public MineColoniesAchievement(final String id, final String name, final int offsetX, final int offsetY, final Item itemIcon, final Achievement parent)
+    {
+        super(id, name, offsetX, offsetY, itemIcon, parent);
+    }
+
+    /**
+     * Parent constructor
+     * 
+     * @param id
+     * @param name
+     * @param offsetX
+     * @param offsetY
+     * @param blockIcon
+     * @param parent
+     */
+    public MineColoniesAchievement(final String id, final String name, final int offsetX, final int offsetY, final Block blockIcon, final Achievement parent)
+    {
+        super(id, name, offsetX, offsetY, blockIcon, parent);
+    }
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)